src/common/utils/setup-helpers.js

```diff
@@ -292,7 +275,7 @@ function getDefaultOptions(options: PartialKPOptionsObject): KPOptionsObject {
   };
   Utils.Object.mergeDeep(defaultOptions, options);
   if (defaultOptions.provider.uiConfId) {
```

Comment:
we want to support also when uiconf id no passed from the app. so can remove this if